java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1752,6 +1752,40 @@ public ColumnVector substring(ColumnVector start, ColumnVector end) {
     }
   }
 
+  /**
+   * Returns a new strings column where target string within each string is removed.
+   *
+   * @param target String to search for within each string.
+   * @return A new java column vector where target string is removed.
+   */
+  public ColumnVector stringReplace(Scalar target) {
```

Comment:
Calling this replace feels a bit odd.  It sounds more like a `stringRemove` to me, but not really sure why we need it anyways.

Reply:
Thanks for pointing it out. I was trying to match it with Spark's  `replace(target)`. I guess we can pass the Scalar empty from higher layer API to this one rather than here. Will remove it from here.